cmd/dptp-controller-manager/main.go

```diff
@@ -27,6 +27,7 @@ import (
 
 	imagev1 "github.com/openshift/api/image/v1"
 
+	clusterpoolreconciler "github.com/openshift/ci-tools/pkg/controller/cluster_pool_reconciler"
```

Comment:
Hm, what's the plan with this reconciler? I noticed that this actually does not reconcile cluster pools, but secrets (based on a presence of cluster pools). Do you plan this to be something larger? At first I actually expected that this will implement the other clusterpool story, the one that would reconcile relesae images in `imageclustersets`.

Can we figure out a better name? `pools_pull_secret_provider` perhaps? (I've tried hard to avoid the word `reconciler` because pretty much every controller is a reconciler)

Reply:
Updated.